site/src/xServices/workspace/workspaceXService.ts

```diff
@@ -551,6 +568,31 @@ export const workspaceMachine = createMachine(
         const message = getErrorMessage(data, "Error getting the applications host.")
         displayError(message)
       },
+      // assignScheduleBannerRef: assign((context) => {
+      //   if (!context.workspace || !context.template) {
+      //     throw Error("Cannot manage workspace schedule without workspace and template.")
+      //   }
+      //   const scheduleContext = {
+      //     workspace: context.workspace,
+      //     template: context.template,
+      //     deadline: getDeadline(context.workspace)
+      //   }
+      //   const scheduleBannerRef = spawn(workspaceScheduleBannerMachine.withContext(scheduleContext), "scheduleBannerMachine")
+      //   return { ...context, scheduleBannerRef }
+      // }),
+      // stopScheduleBannerRef: (context) => {
+      //   context.scheduleBannerRef?.stop && context.scheduleBannerRef.stop()
+      // },
+      // clearScheduleBannerRef: assign({
+      //   scheduleBannerRef: (_) => undefined
+      // }),
```

Comment:
Should we remove these?

Reply:
yes thank you! Makes me realize - I tried both ways, spawning and invoking, and I wrote that I spawned but I really invoked! Haha, you can see why, invoking requires less code.